README.md

```diff
@@ -176,25 +185,42 @@ When the Suite is run, several output files are generated that contain
 data describing the run. The file names start with the file prefix 
 provided via a command line option in the output directory, also specified
 on the command line. If no such options are provided, files will be located 
-in the current directory and be named `RAJAPerf-*`.
+in the current run directory and be named `RAJAPerf-*`, where '*' is a string
+indicating the contents of the file.
 
-Currently, there are up to four files generated:
+Currently, there are five files generated:
 
 1. Timing -- execution time (sec.) of each loop kernel and variant run
 2. Checksum -- checksum values for each loop kernel and variant run to ensure they are producing the same results
 3. Speedup -- runtime speedup of each loop kernel and variant with respect to a reference variant. The reference variant can be set with a command line option. If not specified, the first variant run will be used as the reference. The reference variant used will be noted in the file.
 4. Figure of Merit (FOM) -- basic statistics about speedup of RAJA variant vs. baseline for each programming model run. Also, when a RAJA variant timing differs from the corresponding baseline variant timing by more than some tolerance, this will be noted in the file with `OVER_TOL`. By default the tolerance is 10%. This can be changed via a command line option.
+5. Kernel -- Basic information about each kernel that is run, which is the same
+for each variant of the kernel that is run.
+
+All output files are text files. Other than the checksum file, all are in 
+'csv' format for easy processing by comoon tools and generating plots.
+
+## Kernel information definitions
 
-The name of each file is indicative of its contents. All files are text files. 
-Other than the checksum file, all are in 'csv' format for easy processing 
-by tools and generating plots.
+The kernel information that will appear on the screen when the Suite is run or
+located in the ''RAJAPerf-kernels.csv'' file includes the following:
+
+1. Kernel name -- Format is group name followed by kernel name, separated by an underscore. 
+2. Feature -- RAJA feature(s) exercised in RAJA variants of kernel, available via a command line option.
```

Comment:
Is this in the kernels cvs?

Reply:
No. It wasn't clear to me how to add that for kernels with more than 1 feature. Any thoughts on this? I could add a separate file with feature info.